feat: separate federal and state lands overlays

kmal808 opened the pull request:
Separating the federal-state overlay into federal-lands & state-lands. Both new data files have been simplified to 10% and had the precision reduced to 0.00001. 

Closes ❌ #38 & #39 

See ⏬ for feature preview 👀 

https://user-images.githubusercontent.com/20919083/208237957-d917ba61-72c1-49bc-a955-0f4eddfd7fbf.mov

kmal808 committed:
chore: removing unused files

kmal808 committed:
Merge branch 'main' of https://github.com/CodeforHawaii/Hawaii-Zoning-Atlas

kmal808 committed:
task: adding separate fed & state land data

kmal808 committed:
task: splitting federalState into separate overlays

kmal808 committed:
feat: separate overlays for fed & state lands

tyliec commented on scripts/map.js:
```diff
@@ -468,7 +471,35 @@ var loadFederalState = function () {
+			weight: 1,
+			spaceWeight: 1,
+			angle: 30,
+			color: '#5cc649',
```
```suggestion
			color: '#b349c6',
```
![Screenshot 2022-12-17 at 6 28 52 PM](https://user-images.githubusercontent.com/15609358/208281530-0474a3ae-4973-4946-ba87-14c7eb2ae6de.png)

Thinking about it some more, we should probably color these differently. Randomly picking a color based off what a random website told me to

https://colorpicker.me/#b349c6

kmal808 replied:
Suggestion committed ➡️  [1df5d29](https://github.com/CodeforHawaii/Hawaii-Zoning-Atlas/pull/47/commits/1df5d294e45ea3139179443f68cb4981fb05ae16)

tyliec reviewed (approved):
1 comment about the color, but otherwise LGTM 👍

kmal808 committed:
kmal/style: changing fed & state land overlay color

Changing per https://github.com/CodeforHawaii/Hawaii-Zoning-Atlas/pull/47#discussion_r1051526331

Will use UI teams color palette once chosen
